commands/command_checkout.go

```diff
@@ -0,0 +1,111 @@
+package commands
+
+import (
+	"github.com/github/git-lfs/git"
+	"github.com/github/git-lfs/lfs"
+	"github.com/github/git-lfs/vendor/_nuts/github.com/spf13/cobra"
+	"os"
+	"os/exec"
+	"sync"
+)
```

Comment:
OK never mind, added goimports to my Sublime save action & see it likes to separate stdlib & others. Do the contributor styles notes need updating?